go/stellar/build.go

```diff
@@ -185,6 +287,16 @@ func BuildPaymentLocal(mctx libkb.MetaContext, arg stellar1.BuildPaymentLocalArg
 					})
 				}
 			}
+			offerAdvancedForm, err := ShowAdvancedSendBanner(mctx, remoter, arg.From, arg.To)
+			if err != nil {
+				return res, err
+			}
+			if offerAdvancedForm {
+				res.Banners = append(res.Banners, stellar1.SendBannerLocal{
+					Level:                     "info",
+					OfferAdvancedSendFormbool: true,
+				})
+			}
```

Comment:
Bump one level in, into the `else`. `Because %v hasn’t set up their wallet yet` should preclude `OfferAdvancedSendForm`.